File: test/cases/record/test_record_multi_fields.pas
program test;
type
  point = record
    x, y, z: integer
  end;
var
  p: point;

begin
  p.x := 1;
  p.y := 2;
  p.z := 3;
  write(p.x);
  write(p.y);
  write(p.z)
end.

File: test/cases/record/test_record_multi_fields.c
#include <stdio.h>
typedef struct {
    int x;
    int y;
    int z;
} point;
point p;


int main(void) {
    p.x = 1;
    p.y = 2;
    p.z = 3;
    printf("%d", p.x);
    printf("%d", p.y);
    printf("%d", p.z);

    return 0;
}
